src/main/java/core/basesyntax/service/DeepCopy.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface DeepCopy {
+    List<FruitTransaction> getDeepCopy(List<FruitTransaction> list);
```

Comment:
The method name `getDeepCopy` could be more descriptive. Consider renaming it to something like `cloneFruitTransactionList` to make it clear that it clones a list of `FruitTransaction` objects.